packages/widget/src/pages/details/details-page/components/tab.tsx

```diff
@@ -16,21 +16,26 @@ type Props = {
   isSelected: boolean;
   onTabPress: () => void;
 } & (
-  | { variant: "earn"; hasPendingRewards?: never }
-  | { variant: "positions"; hasPendingRewards: boolean }
+  | { variant: "earn"; pendingRewardsCount?: never }
+  | { variant: "positions"; pendingRewardsCount?: number }
```

Comment:
I'd call this pendingActionsCount -> Has pending actions etc. Since it's not only related to rewards